packages/manager/cypress/e2e/core/placementGroups/create-linode-with-placement-groups.spec.ts

```diff
@@ -0,0 +1,210 @@
+import {
+  mockAppendFeatureFlags,
+  mockGetFeatureFlagClientstream,
+} from 'support/intercepts/feature-flags';
+import { authenticate } from 'support/api/authentication';
+import { makeFeatureFlagData } from 'support/util/feature-flags';
+import { mockGetAccount } from 'support/intercepts/account';
+import {
+  accountFactory,
+  linodeFactory,
+  placementGroupFactory,
+} from 'src/factories';
+import { regionFactory } from 'src/factories';
+import { ui } from 'support/ui/';
+import { cleanUp } from 'support/util/cleanup';
+import { mockCreateLinode } from 'support/intercepts/linodes';
+import { mockGetRegions } from 'support/intercepts/regions';
+import {
+  mockCreatePlacementGroup,
+  mockGetPlacementGroups,
+} from 'support/intercepts/placement-groups';
+import { randomString } from 'support/util/random';
+import { CANNOT_CHANGE_AFFINITY_TYPE_ENFORCEMENT_MESSAGE } from 'src/features/PlacementGroups/constants';
+
+import type { Region } from '@linode/api-v4';
+import type { Flags } from 'src/featureFlags';
+
+const mockAccount = accountFactory.build();
+const mockRegions: Region[] = [
+  regionFactory.build({
+    capabilities: ['Linodes', 'Placement Group'],
+    id: 'us-east',
+    label: 'Newark, NJ',
+    country: 'us',
+  }),
+  regionFactory.build({
+    capabilities: ['Linodes'],
+    id: 'us-central',
+    label: 'Dallas, TX',
+    country: 'us',
+  }),
+];
+
+authenticate();
+describe('Linode create flow with Placement Group', () => {
+  beforeEach(() => {
+    cleanUp('linodes');
```

Comment:
```suggestion
describe('Linode create flow with Placement Group', () => {
  beforeEach(() => {
```
We don't have to call `authenticate` and `cleanUp` here since this test isn't firing off any API requests or actually creating Linodes!